src/diagnostics/radiation_diagnostics.jl

```diff
@@ -58,12 +58,12 @@ function compute_rsu!(
     FT = eltype(state)
     if isnothing(out)
         return RRTMGPI.array2field(
-            FT.(cache.radiation_model.face_sw_flux_up),
+            FT.(cache.atmos.radiation_mode.face_sw_flux_up),
```

Comment:
```suggestion
            FT.(cache.radiation.radiation_model.face_sw_flux_up),
```
And for all the changes below. The change here should be the same as the one in line 23?